rental.rb

```diff
@@ -1,19 +1,30 @@
-require_relative './book'
+# require_relative './book'
+# require_relative './person'
 
 class Rental
-  attr_accessor :date, :person, :book, :rentals
+  attr_accessor :date, :person
+  attr_reader :book
 
   def initialize(date, person, book)
-    @rentals = []
     @date = date
 
     @person = person
-    person.rentals << self unless @person.rentals.include?(self)
+    @person.rentals << self unless @person.rentals.include?(self)
 
     @book = book
-    book.rentals << self unless @book.rentals.include?(self)
+    @book.rentals << self unless @book.rentals.include?(self)
   end
 
+  # def person=(person)
+  #   @person = person
+  #   @person.rentals << self unless @person.rentals.include?(self)
+  # end
+
+  # def book=(book)
+  #   @book = book
+  #   @book.rentals << self unless @person.rentals.include?(self)
+  # end
+
```

Comment:
- [ ] [OPTIONAL] Kindly, remove the commented code from your codebase. This will make your code clean.